Userland/Libraries/LibWeb/Layout/FormattingContext.h

```diff
@@ -107,6 +107,12 @@ class FormattingContext {
         CSSPixels right { 0 };
     };
 
+    struct SpaceUsedAndEdgeForFloats {
```

Comment:
What is an `Edge` in this context?

Reply:
It's the right edge coordinate of the left-side / right-side floats, respectively.